swlor2_2da/appearance.2da

```diff
@@ -10220,4 +10220,42 @@
 10216   "[SWLOR] Ship - Alien Gunship "                                  ****         c_nogu                       c_nogu              ****             N           F           1             1                 1                1                NORM       1.0        2.0       0.3        0.5           1        0.3       1.8            H              1              23           1         1         po_zo_war01       3              9                33             55             1           60           30           head_g            5          1     
 10217   "[SWLOR] Ship - Alien Battleship "                               ****         c_nobs                       c_nobs              ****             N           F           1             1                 1                1                NORM       1.0        2.0       0.3        0.5           1        0.3       1.8            H              1              23           1         1         po_zo_war01       3              9                33             55             1           60           30           head_g            5          1  
 10218   "[SWLOR] Ship - Armored Transport "                              ****         c_fl_artr                    c_fl_artr           ****             N           F           1             1                 1                1                NORM       1.0        2.0       0.3        0.5           1        0.3       1.8            H              1              23           1         1         po_zo_war01       3              9                33             55             1           60           30           head_g            5          1             
-10219   "(Dynamic) Ewok"                                                 ****         Character_model              D                   default          R           P           1             1                 1.5             1.5               NORM       1.06       2.12      0.3        0.5           1.5      0.3       1.4            H              1              169          1         1         ****              3              9                0              0              1           60           30           head_g            6          1            
```

Comment:
The new entry for Ewoks is getting removed. Please copy/paste it over to your file to make sure we don't lose this change.